test/bridge/slippages-test.ts

```diff
@@ -0,0 +1,65 @@
+// BEFORE: :file:`src/bridge/slippages.ts`
+// % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
+//    72.3 |    54.54 |      50 |    72.3 | 35-37,44-45,52-64
+// AFTER: :file:`src/bridge/slippages.ts`
+// % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
+//     100 |    85.71 |     100 |     100 | 28,32,36                    
+
+
+import { BigNumber } from "ethers";
```

Comment:
Nitpick: I like to enforce explicit ethers imports for any/all things; `BigNumber` should be imported from `@ethersproject/bignumber`